controller/user-controller.js

```diff
@@ -0,0 +1,16 @@
+const userService = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname} =req.body;
```

Comment:
- JavaScript는 일반적으로 띄어쓰기를 구분할 때 camelCase로 작성합니다.
- 추가로 skinType도 회원가입을 할 때 받는 데이터입니다. 해당 데이터도 받아오도록 수정해주세요.
- 필수로 데이터베이스에 저장되어야 하는 데이터가 정상적으로 request 객체에 포함되지 않았을 경우 예외처리를 해주세요.
`req` 객체 내부에서 특정 `key`에 대한 `value`(데이터)가 없는 경우 `Key Error`를 반환하도록 작성해주세요.
이 때 skinType은 null 값을 허용토록 스키마를 설계했기 때문에 반드시 예외처리를 하지 않아도 될 것 같습니다.

```suggestion
    const {email, password, firstName, lastName, skinType} =req.body;
    
    if ( !email || !password || !firstName || !lastName) {
      throw new Error("Key Error!");
    }
```